src/pyfile_to_module/pyfile_to_module.py

```diff
@@ -8,9 +8,6 @@
 
 """
 
-__author__ = 'evansde77@gmail.com'
-__version__="0.0.1"
```

Comment:
yeah, its in the __init__ module. I need to make cirrus play nicely with adding it to other scripts without upsetting git.